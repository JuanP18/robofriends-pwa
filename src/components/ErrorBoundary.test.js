import React from 'react';
import {shallow} from 'enzyme';
import ErrorBoundary from './ErrorBoundary';

describe('Error Boundary Component', () => {
  let wrapper;
  beforeEach(() => {
      wrapper = shallow(<ErrorBoundary />)
      wrapper.state({hasError: false});
  })
  it('expect to render ErrorBoundary component' , () => {
      expect(wrapper).toMatchSnapshot();
  });
  it('expect to render ErrorBoundary message' , () => {
    wrapper.state({hasError: true});
});
});
