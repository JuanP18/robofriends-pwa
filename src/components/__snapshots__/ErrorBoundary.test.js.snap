// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Boundary Component expect to render ErrorBoundary component 1`] = `""`;
